tests/models/codegen/test_modeling_codegen.py

```diff
@@ -150,35 +150,6 @@ def get_config(self):
             rotary_dim=self.rotary_dim,
         )
 
-    def prepare_config_and_inputs_for_decoder(self):
```

Comment:
`prepare_config_and_inputs_for_decoder` is used in `encoder-decoder` models and on decoder-only models tested with our `EncoderDecoder` classes.

In the deleted cases in the PR, it was never used -- I suspect it was a leftover copy-paste